fabtests/pytest/common.py

```diff
@@ -8,9 +8,12 @@ def has_cuda(ip):
 
 def has_hmem_support(cmdline_args, host_type, ip):
```

Comment:
You can remove host_type and ip since they're no longer being used. Does UnitTest run on both clietn and server?

Reply:
Removed the stale arguments. Hmm it seems the unit test will only run on the first node. I need to fix this issue.